src/blocks/remote-data-empty-result/render.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types = 1);
+
+use RemoteDataBlocks\Editor\DataBinding\BlockBindings;
+// Global variables provided by WordPress for block rendering:
+// $attributes (array): The block attributes.
+// $content (string): The block default content.
+// $block (WP_Block): The block instance.
+
+$should_render_empty_result = BlockBindings::should_render_empty_result( $block );
+
+// Skip the rendering if the block's results are not empty.
+if ( ! $should_render_empty_result ) {
```

Comment:
Same thing as the JS, skip if the results are not empty otherwise show the empty message which can be customized